web/l10n/en.json

```diff
@@ -232,6 +232,8 @@
     "mapTitle": "Map",
     "mapLayer": "Map Layer",
     "mapCustom": "Custom Map",
+    "mapCustomXyzFormat": "Custom (XYZ)",
```

Comment:
I would keep original string instead. We can potentially change text, but I don't see any reason for adding a new string.

Reply:
I agree. Doing it the way you suggest. Thanks.